arm-web/2016-08-01/swagger/WebApps.json

```diff
@@ -13409,255 +13420,488 @@
     "SiteConfig": {
```

Comment:
Looks like the new model SiteConfigResource replaces this model, if that is the case can we remove this duplicate model?